ui/app/hooks/useCancelTransaction.js

```diff
@@ -17,7 +17,7 @@ import { getConversionRate, getSelectedAccount } from '../selectors'
  */
 export function useCancelTransaction (transactionGroup) {
   const { primaryTransaction, initialTransaction } = transactionGroup
-  const gasPrice = primaryTransaction.txParams?.gasPrice
+  const gasPrice = primaryTransaction.txParams?.gasPrice < 0 ? 0 : primaryTransaction.txParams?.gasPrice
```

Comment:
Isn't `txParams.gasPrice` a string? If so this comparison wouldn't work.

Reply:
Good point! Thanks